locust/static/locust.js

```diff
@@ -189,3 +216,26 @@ function updateExceptions() {
     });
 }
 updateExceptions();
+
+function createEndpointCharts(chartKey, callback) {
+  if(!endpointResponseTimeCharts[chartKey]) {
+    title = "Average Response Times"
```

Comment:
I think this not the average, just response times

Reply:
It's average, the one that using total is response time